src/ring_theory/jacobson.lean

```diff
@@ -31,6 +36,9 @@ Let `R` be a commutative ring. Jacobson Rings are defined using the first of the
 
 * `is_jacobson_iff_Inf_maximal` is the equivalence between conditions 1 and 2 above.
 
+* `is_jacobson_of_surjective` says that if `R` is a jacobson ring and `f : R →+* S` surjective,
+then `S` is also a jacobson ring
```

Comment:
I added some spaces since GitHub's markdown parser doesn't seem to like the current formatting.
```suggestion
* `is_jacobson_of_surjective` says that if `R` is a Jacobson ring and `f : R →+* S` is surjective,
  then `S` is also a Jacobson ring
```